### DISCUSSION TITLE
posts/NewHomeLab_2_5Gbit/

### DISCUSSION THREAD

**Discussion by giscus** [marcmylemans/marcmylemans.github.io#2] [Announcements]
# posts/NewHomeLab_2_5Gbit/

Enhancing Connectivity - The 2.5 Gbit Network Upgrade

https://mylemans.online/posts/NewHomeLab_2_5Gbit/

**Comment by marcmylemans** [comment]
3D printed files i've used to install the second network adapter:
https://www.thingiverse.com/thing:5287455
